README.md

```diff
@@ -20,9 +20,25 @@
   </h3>
 </div>
 
-## Installation
+## Pre-requisites
+The current version of CLI is all-in-one package -- it is used both for production and testing. As a result, even if you
+need CLI only for the token transfer you need to install all testing dependencies. This will be changed in the future.
 
-TODO: Need to publish it to npm.
+If you want to run local Ethereum network you would need to install [ganache-cli](https://www.npmjs.com/package/ganache-cli) globally.
+
+You would also need to install resources needed to compile nearcore (in the future this will only be required for the testing CLI):
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+source $HOME/.cargo/env
+rustup default stable
+rustup target add wasm32-unknown-unknown
```

Comment:
rustup would automatically install nightly, but it would not automatically install nightly's wasm32-unknown-unknown, so this is good to prevent user compile contracts with nightly rust